one_fm/hiring/doctype/onboard_employee/onboard_employee.py

```diff
@@ -246,8 +246,11 @@ def create_user_and_permissions(self):
 			user.first_name = self.employee_name
 			user.email = self.company_email
 			user.role_profile_name = self.role_profile
-			user.save(ignore_permissions = True)
 			employee = frappe.get_doc('Employee', self.employee)
+			
+			user.username = employee.employee_name
```

Comment:
`username` is unique, there may be chance to have same employee name for different employee.
It is better to set `employee_id` from employee as `username` in user